pkg/deploy/elbv2/load_balancer_manager.go

```diff
@@ -161,7 +161,9 @@ func (m *defaultLoadBalancerManager) updateSDKLoadBalancerWithSubnetMappings(ctx
 	for _, az := range sdkLB.LoadBalancer.AvailabilityZones {
 		currentSubnets.Insert(awssdk.StringValue(az.SubnetId))
 	}
-	if desiredSubnets.Equal(currentSubnets) {
+	desiredIPAddressType := string(*resLB.Spec.IPAddressType)
```

Comment:
Hey, I believe we need to check whether IPv6Address is updated rather than IPAddressType.  Also,  While checking IPv6Address, make sure it won't impact ingress behavior as I think the ALB does not support assign static ip.  `resLB.Spec.Type != elbv2model.LoadBalancerTypeApplication`